frontend/pages/[resourceType]/[id].tsx

```diff
@@ -0,0 +1,74 @@
+import { Prism } from '@mantine/prism';
+import { Divider, Group, ScrollArea, Stack, Tabs, Text } from '@mantine/core';
+import { GetServerSideProps } from 'next';
+import { useRouter } from 'next/router';
+import BackButton from '../../components/BackButton';
+
+/**
+ * Component which displays the JSON/ELM/CQL/narrative content of an individual resource using
+ * Mantine tabs
+ * @returns JSON/ELM/CQL/narrative content of the individual resource in a Prism component
+ */
+export default function ResourceIDPage({ jsonData }: { jsonData: fhir4.Measure | fhir4.Library }) {
+  const router = useRouter();
+  const { resourceType, id } = router.query;
+  return (
+    <div>
+      <Stack spacing="xs">
+        <div
+          style={{
+            float: 'left'
+          }}
+        >
+          <Group>
+            <BackButton />
+            <Text size="xl" weight={700} color="gray">
+              {resourceType}/{id}
+            </Text>
+          </Group>
+        </div>
+        <Divider my="sm" style={{ paddingBottom: '5px' }} />
+        <Tabs variant="outline" defaultValue="json">
+          <Tabs.List>
+            <Tabs.Tab value="json">JSON</Tabs.Tab>
+            <Tabs.Tab value="elm" disabled>
+              ELM
+            </Tabs.Tab>
+            <Tabs.Tab value="cql" disabled>
+              CQL
+            </Tabs.Tab>
+            <Tabs.Tab value="narrative" disabled>
+              Narrative
+            </Tabs.Tab>
+          </Tabs.List>
+
+          <Tabs.Panel value="json" pt="xs">
+            <ScrollArea>
+              <Prism
+                language="json"
+                data-testid="prism-page-content"
+                colorScheme="light"
+                style={{ maxWidth: '78vw', height: '80vh', backgroundColor: '#FFFFFF' }}
```

Comment:
Is there a reason we're restricting this to `78vw` and `80vh`? I don't see a reason why for now the prsim viewer can't just take up the whole screen, which you could accomplish by just deleting this whole style prop.

The content is scrollable and seems fine. This style as is adds a lot of whitespace to the right where there will never really be content at least right now